samza-core/src/main/scala/org/apache/samza/config/JobConfig.scala

```diff
@@ -81,6 +81,14 @@ object JobConfig {
   val PROCESSOR_ID = "processor.id"
   val PROCESSOR_LIST = "processor.list"
 
+  // Represents the store path for non-changelog stores which are persisted in the disk and potentially cleaned up
```

Comment:
Remove "persisted in the disk", it's unrelated to changelogging. Also, what does potentially mean here?